docs/01-overview/main-areas/service-management/smgt-99-useful-links.md

```diff
@@ -21,18 +21,18 @@ If you're interested in learning more about the Service Management area, follow
   - ["Found more that one class" error](../../../04-operation-guides/troubleshooting/smgt-02-id-conflict.md)
 
```

Comment:
The link above to `Register a broker in Service Catalog` (line 10) is broken. 
The ending should be `smgt-03-sc-broker-registration.md`.